app/modules/intelligence/tools/confluence_tools/confluence_client.py

```
@@ -40,7 +42,7 @@ async def check_confluence_integration_exists(user_id: str, db: Session) -> bool
         )
         return integration is not None
     except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)